target/linux/ipq40xx/patches-6.6/700-net-ipqess-introduce-the-Qualcomm-IPQESS-driver.patch

```diff
@@ -1201,6 +1196,16 @@ Signed-off-by: Maxime Chevallier <maxime.chevallier@bootlin.com>
 +	SET_NETDEV_DEV(netdev, &pdev->dev);
 +	platform_set_drvdata(pdev, netdev);
 +
++	err = of_get_ethdev_address(np, netdev);
++	if (err) {
++		dev_info(&pdev->dev, "of_get_ethdev_address: %d\n", err);
```

Comment:
I would drop this print, or make ti something like "Using DT MAC address" and demote it to `dev_dbg`

Reply:
Yes, planned to make this `dev_dbg`, maybe reads better: "Failed to get MAC address from DT: %d\n", err
Don't think we need to see it for every device that does not (or has problem) retrieving address from DT, just wanted to see if I could get someone with an MR33 to run it first.

Reply:
Yeah, that sounds good.